### PR TITLE
\hyperlink{block_nonce}{nonce} + related links

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -1197,7 +1197,7 @@ \subsection{Mining Proof-of-Work} \label{ch:pow}
...
+Two directions exist for ASIC resistance; firstly make it sequential memory-hard, i.e. engineer the function such that the determination of the \hyperlink{block nonce}{nonce} requires a lot of memory and bandwidth such that the memory cannot be used in parallel to discover multiple nonces simultaneously. The second is to make the type of computation it would need to do general-purpose; the meaning of ``specialised hardware''  for a general-purpose task set is, naturally, general purpose hardware and as such commodity desktop computers are likely to be pretty close to ``specialised hardware'' for the task. For Ethereum 1.0 we have chosen the first path.
```
Comment (pirapira) [inline Paper.tex]: The link does not work.  Maybe the name of the target should include some underscores.

Comment (jamesray1) [reply Paper.tex]: Fixed.

**Comment by pirapira** [comment]
https://travis-ci.org/ethereum/yellowpaper/builds/343155240?utm_source=github_status&utm_medium=notification

```
pdfTeX warning (ext4): destination with the same identifier (name{block}) has b
een already used, duplicate ignored
<to be read again> 
                   \relax 
l.324 \subsection{The Block} \linkdest{block}
                                             {}
Overfull \vbox (4.76pt too high) has occurred while \output is active [4]
! Undefined control sequence.
l.345 ...ce]\linkdest{block_nonce_H__n}{}\linkdset
                                                  {block_nonce} A 64-bit has...
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on build/Paper.log.
```

**Comment by jamesray1** [comment]
This should be fine to merge.

File: Paper.tex [hunk 2]
```diff
@@ -342,7 +342,7 @@ \subsection{The Block}\linkdest{block}\label{subsec:The_Block}
 \item[timestamp]\linkdest{block_timestamp_word_def_H__s}{} A scalar value equal to the reasonable output of Unix's time() at this block's inception; formally \hyperlink{block_timestamp_H__s}{$H_{\mathrm{s}}$}.
 \item[extraData]\linkdest{block_extraData_H__x}{} An arbitrary byte array containing data relevant to this block. This must be 32 bytes or fewer; formally $H_{\mathrm{x}}$.
 \item[mixHash]\linkdest{mixHash_H__m}{} A 256-bit hash which, combined with the nonce, proves that a sufficient amount of computation has been carried out on this block; formally $H_{\mathrm{m}}$.
-\item[nonce]\linkdest{block_nonce_H__n}{}\linkdest{block_nonce} A 64-bit value which, combined with the mix-hash, proves that a sufficient amount of computation has been carried out on this block; formally $H_{\mathrm{n}}$.
+\item[nonce]\linkdest{block_nonce_H__n}{}\linkdest{block_nonce} A 64-bit hash which, combined with the mix-hash, proves that a sufficient amount of computation has been carried out on this block; formally \hyperlink{H__n}{$H_{\mathrm{n}}$}.
```
Comment (pirapira) [inline Paper.tex]: The nonce is not a hash.

**Pull request merged**